StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/VerticalPaymentMethodListViewController.swift

```diff
@@ -114,7 +114,7 @@ class VerticalPaymentMethodListViewController: UIViewController {
             let rowButton = RowButton.makeForPaymentMethodType(
                 paymentMethodType: paymentMethodType,
                 subtitle: Self.subtitleText(for: paymentMethodType),
-                savedPaymentMethodType: savedPaymentMethod?.type,
+                hasSavedCard: savedPaymentMethod?.type == .card, // TODO(RUN_MOBILESDK-3708) 
```

Comment:
By passing in `savedPaymentMethod?.type == .card` we end up showing "New Card", if we want to do this work later as part of 3708, and continue to keep this as just "Card", seems like we just need to pass `false` into here?

Reply:
Nevermind - this will maintain the existing behavior, and the jira is actually fix an existing bug where we show "card" even if we have a saved card that isn't selected.